src/processing.integration-test.ts

```diff
@@ -31,47 +36,55 @@ describe("processSpace", () => {
   });
 
   it("should process proposals correctly", async () => {
-    const notifyFn = spy();
+    const processProposalsFn = spy();
+    const processAnswersFn = spy();
     const emitter = new EventEmitter();
 
-    const toBlock = 19475121n;
+    const toBlock = oneInchProposalBlockNumber + 1n;
 
     const result = await fn({
       space,
       blockNumber: toBlock,
       calculateBlockRangeFn: () => ({
-        fromBlock: 19475120n,
         toBlock,
+        fromBlock: oneInchProposalBlockNumber,
       }),
       emitter,
-      notifyFn,
       updateSpaceFn: updateSpace,
+      processProposalsFn,
+      processAnswersFn,
     });
 
     expect(result.lastProcessedBlock).to.equal(toBlock);
-    expect(notifyFn.calledOnce).to.be.true;
-    const call = notifyFn.firstCall.firstArg;
-    expect(call).to.deep.equal({
-      type: EventType.PROPOSAL_QUESTION_CREATED,
-      space,
-      event: {
+
+    expect(processProposalsFn.calledOnce).to.be.true;
+    const processProposalsCall = processProposalsFn.firstCall;
+    expect(processProposalsCall.args[0]).to.deep.equal(space);
+    expect(processProposalsCall.args[1]).to.deep.equal([
+      {
         proposalId: "0x791b3d71ea14497d3b8e756479f6f126e08b44351bfab904834c56b1ccf0479a",
-        questionId: "0xebf5b601fedfaa5562a03590e9ac8be937cc070a131443af01948a7eda6dfabf",
```

Comment:
are this random hashes? can you add comment saying it is random or specifics of why these hashes.